packages/buidler/contracts/NiftyInk.sol

```diff
@@ -140,30 +133,17 @@ contract NiftyInk is BaseRelayRecipient, Ownable, SignatureChecker {
       return _setPrice(_ink.id, price);
     }
 
-    function ownershipRequest(string memory fileUrl, address artist, address newArtist, bytes memory signature) public {
+    function transferOwnership(string memory fileUrl, address artist, address payable newArtist, bytes memory signature) public {
```

Comment:
Why payable?

Reply:
@ironsoul0 because of the struct
```
    struct File {
      uint256 id;
      address payable artist;
      string jsonUrl;
      string fileUrl;
      uint256 limit;
      bytes signature;
      uint256 price;
      Counters.Counter priceNonce;
    }
```